scripts/scripts.js

```diff
@@ -31,6 +32,31 @@ function buildHeroBlock(main) {
   }
 }
 
+function buildautoblogBlock(main) {
+  const h1 = main.querySelector('h1');
+  if (h1 && (getMetadata('template') === 'blog')) {
+    console.log('This is valid for Autoblogging');
+    const authorcontent = getMetadata('author');
+    const publishedcontent = getMetadata('published');
+    const readcontent = getMetadata('readtime');
+    const section = document.createElement('div');
+    const elems = [authorcontent, publishedcontent, readcontent];
+    const subsection1 = document.createElement('div');
+    elems.forEach((x, i) => { const col = document.createElement('div'); col.textContent = x; subsection1.appendChild(col); });
+    section.appendChild(subsection1);
+    subsection1.after(h1);
+    const updatedcontent = getMetadata('updated');
```

Comment:
There's no metadata for "updated", remember -- this is going to just be a manual text addition to the body copy if the author decides to use it. Only the published date is autoblocked.